packages/victory-core/src/victory-util/wrapper.tsx

```diff
@@ -298,7 +298,7 @@ export function getAllEvents(props) {
   return events || [];
 }
 
-export function getColor(calculatedProps, child, index) {
```

Comment:
@carbonrobot it looks like `getColorScale` from `victory-util/style.ts` is also a public method which causes breaking changes, right? 

If so, would the best way to go about this to just create a similar helper method that's only used within the `demos` directory that determines color scale? However, that will mean that Victory won't be able to take full advantage of the new theme until these breaking changes are reintroduced.

Reply:
Ahh I see what you mean. I can adjust these methods so that I'm also broadening them. It does result in a lint error around 3 being the max params, but I can ignore that inline if that sounds good to you? @carbonrobot